src/backend/core/tests/team_accesses/test_api_team_accesses_list.py

```diff
@@ -203,3 +203,82 @@ def test_api_team_accesses_list_authenticated_constant_numqueries(
 
     assert response.status_code == 200
     assert response.json()["count"] == 21
+
+
+def test_api_team_accesses_list_authenticated_ordering():
+    """Team accesses can be ordered by "role"."""
+
+    user = factories.UserFactory()
+    factories.IdentityFactory(user=user, is_main=True)
+
+    team = factories.TeamFactory()
+    models.TeamAccess.objects.create(team=team, user=user)
+
+    # create 20 new team members
+    for _ in range(20):
+        extra_user = factories.IdentityFactory(is_main=True).user
+        factories.TeamAccessFactory(team=team, user=extra_user)
+
+    client = APIClient()
+    client.force_login(user)
+
+    response = client.get(
+        f"/api/v1.0/teams/{team.id!s}/accesses/?ordering=role",
+    )
+    assert response.status_code == 200
+    assert response.json()["count"] == 21
+
+    results = [team_access["role"] for team_access in response.json()["results"]]
+    assert sorted(results) == results
+
+    response = client.get(
+        f"/api/v1.0/teams/{team.id!s}/accesses/?ordering=-role",
+    )
+    assert response.status_code == 200
+    assert response.json()["count"] == 21
+
+    results = [team_access["role"] for team_access in response.json()["results"]]
+    assert sorted(results, reverse=True) == results
+
+
+@pytest.mark.parametrize("ordering_fields", ["name", "email"])
+def test_api_team_accesses_list_authenticated_ordering_user(ordering_fields):
```

Comment:
I feel like there's a smart way to factorize those two tests of same purpose using 

```
@pytest.mark.parametrize(
    "ordering_fields, method", 
    [
        ("role", "role"), 
        ("name", '["user"]["name"]'), 
        # ("email", "user__email")
    ])
```
It does raise a key error because of the presence absence of `["user"]` but I feel like a `if` just before the results would allow for a nice factorisation.

Reply:
Actually, I prefer having dumb test, instead of complicated tests parametrized and with if that change their logic. In this situation I understand your point, if you really need this factorization I'll do it

Reply:
We should not chase test factorization

Reply:
+1 on the opinion that this is too smart and makes it much harder to understand the test and each of the 2 contexts you are trying to test because I should understand the "why" by reading the docstring.